### ISSUE TITLE
Is there a way to destroy an instance without a reference to the MediumEditor object?

### ISSUE THREAD

**Issue by brunomperes** [yabwe/medium-editor#590]

**Comment by nmielnik** [comment]
Currently there isn't a way to get the instance of medium-editor without storing a reference yourself.

This is a good idea as it sounds like a useful feature however...it is definitely worth a pull request :smile:

**Comment by nmielnik** [comment]
Each element monitored by MediumEditor now has a `medium-editor-index` attribute on it.  This index corresponds to the index in the global `window._mediumEditors` Array that the MediumEditor instance lives.

So, give an element maintained by MediumEditor, you could do this:

```js
// assume 'element' is a `<div>` that was passed into MediumEditor before.
var index = parseInt(element.getAttribute('medium-editor.index'));
var editor = window._mediumEditors[index]; // This gives you a reference to the editor
if (editor) {
  editor.destroy();
}
```

I think this should take care of what you're asking for.  Perhaps a static helper method on the global MediumEditor object would be a good idea?  Something like `MediumEditor.getInstanceByElement(element)`?  Would make a great contribution to the project if you find it useful!

**Comment by anilmaurya** [comment]
Now Medium Editor have option to destroy instance without storing reference.

`window._mediumEditors` stores medium editor instances.

To select an instance on DOM element with class '.editable'

      var editor = window._mediumEditors.filter(function(ele, index){return (ele && ele.origElements == '.editable')})[0]

To destroy instance

      editor.destroy()

@nmielnik you can close this issue.

**Comment by nmielnik** [comment]
Thanks @anilmaurya !

**Comment by brunomperes** [comment]
To anyone interested, this is available since v5.0.0-rc.1 via #664 https://github.com/yabwe/medium-editor/pull/664/commits/f692918a9847350b3c427f59d90a8d23edffe41d#diff-c1a8a21c933e1f48d90be645db83630fR162
